Index: main.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package main\r\n\r\nimport (\r\n\t\"fmt\"\r\n)\r\n\r\nfunc main() {\r\n\tfmt.Println(\"start\")\r\n\tcnt := 10\r\n\tfor mask := 0; mask < 1<<cnt; mask++ {\r\n\t\tfor i := 0; i < cnt; i++ {\r\n\t\t\tfmt.Printf(\"%b >> %v  = %b\\n\", mask, i, mask>>i)\r\n\t\t}\r\n\t}\r\n\tfmt.Println(\"done!\")\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.go b/main.go
--- a/main.go	
+++ b/main.go	
@@ -6,11 +6,12 @@
 
 func main() {
 	fmt.Println("start")
-	cnt := 10
-	for mask := 0; mask < 1<<cnt; mask++ {
-		for i := 0; i < cnt; i++ {
-			fmt.Printf("%b >> %v  = %b\n", mask, i, mask>>i)
-		}
+	s := "abcABC123"
+	//for idx, v := range s {
+	//	fmt.Println(idx, v, s[idx], reflect.TypeOf(v), reflect.TypeOf(s[idx]), string(v))
+	//}
+	for _, v := range []byte(s) {
+		fmt.Println(v)
 	}
 	fmt.Println("done!")
 }
